Localide/Classes/Localide.swift

```diff
@@ -150,11 +150,13 @@ public final class Localide: NSObject {
         completion: LocalideUsageCompletion?
     ) {
         if app == .copilot {
+            var openStatus: Bool = false
             if let urlString = customUrlsPerApp[app] {
-                _ = LocalideMapApp.launchAppWithUrlString(urlString)
+                openStatus = LocalideMapApp.launchAppWithUrlString(urlString)
             } else if let copilotWithoutParams = LocalideMapApp.prefixes[app] {
-                _ = LocalideMapApp.launchAppWithUrlString(copilotWithoutParams)
+                openStatus = LocalideMapApp.launchAppWithUrlString(copilotWithoutParams)
             }
+            completion?(app, fromMemory, openStatus)
```

Comment:
Can we unit test this?

Reply:
https://bringg.atlassian.net/browse/BRNGG-29255